src/index.js

```diff
@@ -4,13 +4,19 @@ const openPr = require('./openPr')
 const release = require('./release')
 const { logError } = require('./log')
 
-module.exports = async function ({ github, context, inputs, packageVersion }) {
+module.exports = async function ({
+  github,
+  context,
+  inputs,
+  packageVersion,
+  packageName,
+}) {
   if (context.eventName === 'workflow_dispatch') {
     return openPr({ context, inputs, packageVersion })
   }
 
   if (context.eventName === 'pull_request') {
-    return release({ github, context, inputs })
+    return release({ github, context, inputs, packageVersion, packageName })
```

Comment:
back to my previous point, if we can assume that the code has access to the source code (it assumes we're doing a checkout in the yaml), then we can grab package name and version here in a much simpler way by loading the package json of the repo. beware of the paths though

Reply:
The requested refactoring is done, the package info is not loaded in the notify issue util